queue/sqs.go

```diff
@@ -36,8 +36,9 @@ type sqsFIFOQueue struct {
 		completed map[string]bool
 		all       map[string]amboy.Job
 	}
-	runner amboy.Runner
-	mutex  sync.RWMutex
+	runner       amboy.Runner
+	retryHandler amboy.RetryHandler
```

Comment:
Will this be a retryable queue?

Reply:
Nope, removed. Only the remoteQueues are going to support it.